xml/art-nvidia-vgpu.xml

```diff
@@ -0,0 +1,1137 @@
+       <option>--dkms</option> option:
+      </para>
+<screen>&prompt.sudo;./NVIDIA-Linux-x86_64-450.55-vgpu-kvm.run --dkms</screen>
+     </tip>
+    </step>
+    <step>
+     <para>
+      When the driver installation is finished, reboot the system:
+     </para>
+<screen>&prompt.sudo;systemctl reboot</screen>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="configure-nvidia-vgpu-verify-driver-installation">
+   <title>Verify the driver installation</title>
+   <procedure>
+    <step>
+     <para>
+      Verify loaded kernel modules:
+     </para>
+<screen>
+&prompt.user;lsmod | grep nvidia
+nvidia_vgpu_vfio       49152  9
+nvidia              14393344  229 nvidia_vgpu_vfio
+mdev                   20480  2 vfio_mdev,nvidia_vgpu_vfio
+vfio                   32768  6 vfio_mdev,nvidia_vgpu_vfio,vfio_iommu_type1
+</screen>
+     <para>
+      The modules containing the <literal>vfio</literal> string are required
+      dependencies.
+     </para>
+    </step>
+    <step>
+     <para>
+      Print the GPU device status with the <command>nvidia-smi</command>
+      command. The output should be similar to the following one:
+     </para>
+<screen>
+&prompt.user;nvidia-smi
++-----------------------------------------------------------------------------+
+| NVIDIA-SMI 470.63       Driver Version: 470.63       CUDA Version: N/A      |
+|-------------------------------+----------------------+----------------------+
+| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
+| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
+|                               |                      |               MIG M. |
+|===============================+======================+======================|
+|   0  NVIDIA A40          Off  | 00000000:31:00.0 Off |                    0 |
+|  0%   46C    P0    39W / 300W |      0MiB / 45634MiB |      0%      Default |
+|                               |                      |                  N/A |
++-------------------------------+----------------------+----------------------+
+
++-----------------------------------------------------------------------------+
+| Processes:                                                                  |
+|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
+|        ID   ID                                                   Usage      |
+|=============================================================================|
+|  No running processes found                                                 |
++-----------------------------------------------------------------------------+
+</screen>
+    </step>
+    <step>
+     <para>
+      Check the sysfs file system. For Volta and earlier GPU cards, new
+      directory <filename>mdev_supported_types</filename> is added, for
+      example:
+     </para>
+<screen>cd /sys/bus/pci/devices/0000\:84\:00.0/mdev_supported_types</screen>
```

Comment:
should be  "00000000\:31\:00.0" in the path

Reply:
```suggestion
<screen>cd /sys/bus/pci/devices/00000000\:31\:00.0/mdev_supported_types</screen>
```